laravel/app/Models/ArticleTag.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class ArticleTag extends Model
```

Comment:
many to many relation between article and tag should not have a separated model, please remove it